src/main/java/kafdrop/controller/AclController.java

```diff
@@ -32,6 +33,7 @@
 
 import java.util.List;
 
+@Tag(name = "acl-controller")
```

Comment:
SpringFox derived a description from the class name. Let's explicitly add one for each controller, like so:
```suggestion
@Tag(name = "acl-controller", description = "ACL Controller")
```